vtStorAtaProtocol/Platform/Linux/ProtocolAtaPassThrough.h

```diff
@@ -30,33 +30,116 @@ namespace vtStor
 namespace Protocol
 {
 
+#define IOCTL_SG_IO_ERROR -1
+
+const U32 SG_IO_INTERFACE_ID             = 'S';
+const U8  SG_ATA_PASS_THROUGH_16         = 0x85;

```

Comment:
Add it to a enum, what shoud I do if I need to issue a new ATA_PASS_THROUGH type?